Source/DafnyDriver/DafnyDriver.csproj

```diff
@@ -1,18 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
From the PR description:

> Relabel DafnyDriver as DafnyDriver.dll instead of Dafny.dll, remove reference to DafnyRuntime
I'll be able to use this entry point to build Dafny source code into the runtimes when implementing https://github.com/dafny-lang/dafny/issues/2390

Why do you need this project? Can't you use DafnyCore instead? Are you planning on setting the output type here to Exe again so you can run it?

Reply:
I'm planning on adding an empty Exe project that references `DafnyDriver`, something like `DafnyWithoutRuntimes`, so I can easily invoke the Dafny compiler from something like a Makefile. I don't know of any way to invoke logic in `DafnyCore` directly, but I'd be happier if there was!